vars/camundaPublish.groovy

```diff
@@ -0,0 +1,19 @@
+def call(String s2sServiceName, String environment, String product) {
+
+  def camundaUrl = "http://camunda-api-${environment}.service.core-compute-${environment}.internal"
+  def s2sUrl = "http://rpe-service-auth-provider-${environment}.service.core-compute-${environment}.internal"
+
+  stageWithAgent('Camunda - Publish BPMN and DMN', product) {
+
+    log.info 'Publish to Camunda'
+
+    functions = libraryResource 'uk/gov/hmcts/pipeline/camunda/publish-camunda-processes.sh'
```

Comment:
Tried something like this ? https://github.com/hmcts/cnp-jenkins-library/commit/170499263ba124234d8f8bd0587dc77f92693c17#diff-2ca73d4bb738d9d336c1c6d0126c0eda9ef2de3f563424d065efe234b6ffc570R3